### PR TITLE
Replace Fall Events in Highlights Section

### PR DESCRIPTION
**Pull request by khxia**
# Changes

Removed fall events and all of our old banners. 
Added winter events on the highlights sections.
- The `curriculum` link for Hack Sprint has yet to be created so will update that when I can.
- All information on the HOTH 9 section is temporary and is reusing last year's assests.

## Type of change

Please delete options that are not relevant.

- [x] Content Update (non-breaking change which updates the content of the website)

# Related Issues

Link the relevant issue here
#278

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,297 +1,297 @@
 module.exports = {
```
Comment (jodymlin) [inline .eslintrc.js:1]: why was this changed?

File: src/data/events/highlights.js [hunk 2]
```diff
@@ -1,49 +1,37 @@
...
-	series led by students who will share their passion for specific fields,
-	topics, or technologies in the hopes of helping others navigate the vast
-	world of tech to find their own passions.
-`;
+const hothDescription = `
+	Hack on the Hill (HOTH 9) is our annual hackathon.
+	This beginner-friendly hackathon is designed to give you a
+	glimpse of a real hackathon. There will be food, workshops,
+	mentors, and fun prizes for the best hacks!
+	`;
```
Comment (jodymlin) [inline src/data/events/highlights.js:15]:
```suggestion
`;
```

File: src/data/events/highlights.js [hunk 3]
```diff
@@ -1,49 +1,37 @@
-const hackschoolDescription = `
-	Come out and learn how to make websites in our beginner-friendly Fall
-	Hackschool series! We will be teaching the basics of web development
-	with React.js, a popular framework of Javascript. No previous experience
-	is necessary!
+const hackSprintDescription = `
+	Come out to ACM Hack's Hack Sprint, a beginner-friendly mobile
+	development workshop series! We'll be covering everything in
+	Swift from building your app from scratch to publishing a fully
+	functional app on the App Store. Our series will end with a c
```
Comment (jodymlin) [inline src/data/events/highlights.js:5]: Change the word break from `c\nhance` to just `chance` on the next line

**Review by jodymlin — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (khxia) [reply .eslintrc.js:1]: This might have been the autoformatter. Honestly don't know

File: src/data/events/highlights.js (hunk 3 above)
Comment (khxia) [reply src/data/events/highlights.js:5]: Again, this was also the formatter. Have no idea why it's doing this.

Comment (nareha) [reply src/data/events/highlights.js]: Does the line pass 79 characters with the full word on that line? Formatter is probably trying to stick to convention for line length. You can probably disable that in the rules somewhere, but I'm not aware of how.

File: .eslintrc.js (hunk 1 above)
Comment (jodymlin) [reply .eslintrc.js:1]: Yes let's keep it as it was before the change

**Review by jodymlin — APPROVED** [review]
